packages/auth/src/providers/cognito/apis/confirmSignIn.ts

```diff
@@ -76,11 +76,11 @@ export async function confirmSignIn(
 			This most likely occurred due to:
 			1. signIn was not called before confirmSignIn.
 			2. signIn threw an exception.
-			3. page was refreshed during the sign in flow.
+			3. page was refreshed during the sign in flow and resumed session has expired.
```

Comment:
```suggestion
			3. page was refreshed during the sign in flow and session has expired.
```

I'd remove the references to "resumable" / "resumed" to as not confuse customers that might be missing context.